core/src/test/java/com/sequenceiq/cloudbreak/controller/validation/stack/StackValidatorTest.java

```diff
@@ -30,6 +33,8 @@
 import com.sequenceiq.cloudbreak.service.rdsconfig.RdsConfigService;
 import com.sequenceiq.cloudbreak.structuredevent.CloudbreakRestRequestThreadLocalService;
 import com.sequenceiq.cloudbreak.validation.ValidationResult;
+import com.sequenceiq.cloudbreak.workspace.model.Tenant;
+import com.sequenceiq.cloudbreak.workspace.model.User;
 
 @RunWith(MockitoJUnitRunner.class)
 public class StackValidatorTest extends StackRequestValidatorTestBase {
```

Comment:
Maybe I overlooked it but could not find a test case checking for the failure condition of `StackValidator.validateVariant()`.